tests/unit/SidebarOrderTest.ts

```diff
@@ -780,7 +784,12 @@ describe('Sidebar', () => {
                     })
 
                     // When a new report is added
-                    .then(() => Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${report4.reportID}`, report4))
+                    .then(() =>
+                        Promise.all([
```

Comment:
@shubham1206agra in order to sync the merges in Onyx, I've used `Promise.all` here. Let me know if that's the correct way 🙏

Reply:
@shubham1206agra AFAIK, there is no `multiSet` alternative (except for lower-level `Onyx.update`) so in order to merge those two values 'at once' (they are done separately but eventual consistency is what we care about), they are wrapped with `Promise.all`. I am not sure, if other methods would have exact same outcome. 

tl;dr; It waits for all the values to be merged successfully so the test can operate on up-to-date values